### ISSUE TITLE
`unmanaged-cluster` list is hard to use with other unix tools

### ISSUE THREAD

**Issue by jpmcb** [vmware-tanzu/community-edition#3326]
## Feature Request

Currently, it's hard to use `unmanaged-cluster list` with other unix tools, like `xargs`

```
$ tanzu unmanaged-cluster list
  NAME        PROVIDER
  my-cluster  kind
  woof        kind
```

If I wanted to pipe this into `xargs` and perform operations on each cluster, I'd have to first to ignore the first line and also ignore the provider column. 

Instead, we should provide some option that _only_ outputs the names of each cluster, making it really easy to work with:

```shell
$ tanzu unmanage-cluster list -o list-names
my-cluster
woof
```

```shell
$ tanzu unmanage-cluster list -o list-names | xargs echo
my-cluster woof
```

This is very similar to what `kind` does:
```shell
$ kind get clusters
my-cluster
woof
```

Which supports easy operations with `xargs` like deleting all clusters:
```shell
$ kind get clusters | xargs -I _ kind delete cluster --name  _
Deleting cluster "my-cluster" ...
Deleting cluster "woof" ...
```

#### TLDR

`unmanaged-cluster` list should provide a "name" only option in support of it being easily workable with other unix tools.

We should also _consider_ making this the default option so that out of the box it's easily used with other command line tools

## Describe alternatives you've considered

I could just use `jq` to get a list of the names from the `--json` output but `jq` can be really tricky and isn't available on most systems by default

**Comment by joshrosso** [comment]
Historically, I've really liked `docker`'s usage of `-q` to list non-table outputs.
